web-ng/src/app/components/form-field-editor/form-field-editor.component.ts

```diff
@@ -98,9 +107,13 @@ export class FormFieldEditorComponent implements OnInit, OnChanges {
   }
 
   ngOnChanges(changes: SimpleChanges) {
-    const type = this.fieldTypes.find(
-      fieldType => fieldType.type === Number(this.type)
-    );
+    const type = !this.cardinality
```

Comment:
Cardinality only applies in when "Select one" or "Select many" is selected, but here it seems it's always evaluated. Is that intentional?

Reply:
I think this is for property passed from parent to child for fields with stored values of "Select one" or "Select many". @gauravchalana1  am I right?

Reply:
A few things seem to be muddling my understanding here:
1. I see we also have `onFieldTypeSelect` defined. If that's the case, why do we also have `ngOnChanges`? I assume it's because @Input can be changed by the parent after the component has been constructed? If so let's add a comment to clarify.
2. Assuming my understanding in (1) correct, we should first check the type provided; if it's anything other than `multiple_choice`, we should ignore cardinality. If it is `multiple_choice`, only then do we care about `cardinality`. Based on this the current logic may work, but it's counterintuitive given the logical hierarchy is field type->cardinality, not cardinality->field type.

Reply:
Hi @gino-m , I have now tried and updated the function as you have suggested. Please review and share your comments. :) ty